### ISSUE TITLE
Closing a modmail with a user that has been banned throws an error 

### ISSUE THREAD

**Issue by JamesZoft** [chamburr/modmail#61]
Hi,

If a member raises a modmail, is banned, and then the modmail is closed afterwards, the line here:

https://github.com/chamburr/modmail/blob/651d15fa714aa198b11b9ec2768c2d9713217976/cogs/core.py#L70

will mean that an error is thrown:

```2021-04-03T15:40:40.214214776Z  [Cluster 1] ERROR:cogs.error_handler:NotFound: 404 Not Found (error code: 10007): Unknown Member (In close)
2021-04-03T15:40:40.214229764Z  [Cluster 1] Traceback:
2021-04-03T15:40:40.214233302Z  [Cluster 1]   File "/usr/local/lib/python3.8/dist-packages/discord/ext/commands/core.py", line 85, in wrapped
2021-04-03T15:40:40.214236122Z  [Cluster 1]     ret = await coro(*args, **kwargs)
2021-04-03T15:40:40.214238968Z  [Cluster 1]   File "/modmail/modmail/cogs/core.py", line 181, in close
2021-04-03T15:40:40.214241777Z  [Cluster 1]     await self.close_channel(ctx, reason)
2021-04-03T15:40:40.214250098Z  [Cluster 1]   File "/modmail/modmail/cogs/core.py", line 70, in close_channel
2021-04-03T15:40:40.214253042Z  [Cluster 1]     member = await ctx.guild.fetch_member(tools.get_modmail_user(ctx.channel).id)
2021-04-03T15:40:40.214256614Z  [Cluster 1]   File "/modmail/modmail/classes/guild.py", line 86, in fetch_member
2021-04-03T15:40:40.214268363Z  [Cluster 1]     data = await self._state.http.get_member(self.id, member_id)
2021-04-03T15:40:40.214271118Z  [Cluster 1]   File "/usr/local/lib/python3.8/dist-packages/discord/http.py", line 250, in request
2021-04-03T15:40:40.214273671Z  [Cluster 1]     raise NotFound(r, data)```

and more importantly, that no logs are output.

**Comment by JamesZoft** [comment]
There probably isn't a great way to totally fix this - I've informed our mods to not ban before closing, but maybe you could stop it from throwing an exception and log whatever you can? :)

**Comment by FeelsBadMan1** [comment]
Hello, This error is happening because the bot cant get the member anymore cause as you told hes banned. This error will not break the bot is just shows a Information. If you consider want to not print that out into the console or create a event that will be triggerd on errors so the bot sending errors within the discordchat instead of the console you could write me back and then i can give you a premade code. im coding in python about a year now and learned a lot where i now can give basic help to others.

**Comment by FeelsBadMan1** [comment]
You also could add a code so its checking if the member exist in the server and if so it continue with the code if not it will return (do nothing) or run a other code. in case you want me to help you you also could ask me at my discordaccount cause there i have a faster responsetime. | The German Destroyer#7500

**Comment by chamburr** [comment]
Hi, sorry for the late response as I overlooked this. Thanks for the bug report!

I will look into the issue for the next release. Meanwhile, a pull request to fix this would be appreciated.

**Comment by ReyBotDev** [comment]
@chamburr You can fix this by adding a check whether the user is now in the server or not.

**Comment by Codeize** [comment]
> @chamburr You can fix this by adding a check whether the user is now in the server or not.

Like he said, he will look into it for the next release, I'm sure he knows well how to fix it.

**Comment by chamburr** [comment]
Fixed in e0b0bce2776634c7ea2d36f2ea642a04d9c20009.
